public/dashboard.js

```diff
@@ -207,6 +292,12 @@ handleLogout = async () => {
     showLoginForm(true);
     localStorage.removeItem("authenticationType");
     clearToken();
+
+    // OIDC needs to redirect which should happen
+    // after the clearToken + localStorage pruning
+    if (authenticationType === "oidc") {
+        handleOidcLogout();
```

Comment:
technically we should have an `await` here too, or remove the `async` decorator on `handleOidcLogout` if it's not needed